website/docs/r/networking_port_v2.html.markdown

```diff
@@ -63,24 +83,26 @@ The following arguments are supported:
 * `fixed_ip` - (Optional) An array of desired IPs for this port. The structure is
     described below.
 
-* `allowed_address_pairs` - (Optional) An IP/MAC Address pair of additional IP
-    addresses that can be active on this port. The structure is described
-    below.
+* `allowed_address_pairs` - (Optional) An array of IP/MAC Address pairs of additional IP
+    addresses that can be active on this port. The structure is described below.
 
 * `value_specs` - (Optional) Map of additional options.
 
 The `fixed_ip` block supports:
 
-* `subnet_id` - (Required) Subnet in which to allocate IP address for
-this port.
+* `subnet_id` - (Required) Subnet in which to allocate IP address for this port.
 
 * `ip_address` - (Optional) IP address desired in the subnet for this port. If
-you don't specify `ip_address`, an available IP address from the specified
-subnet will be allocated to this port.
+    you don't specify `ip_address`, an available IP address from the specified
+    subnet will be allocated to this port.
 
 The `allowed_address_pairs` block supports:
 
-* `ip_address` - (Required) The additional IP address.
+* `ip_address` - (Required) The additional IP address. The value can be an IP Address or a CIDR,
+    and can not be *0.0.0.0*. A server connected to the port can send a packet with source address
```

Comment:
@ShiChangkuo did you mean `can not be *0.0.0.0/0*` as a CIDR (with the trailing /0) as in https://support.huaweicloud.com/intl/en-us/api-vpc/vpc_port01_0006.html#vpc_port01_0006__table2033432574917 ?
> You cannot set it to 0.0.0.0/0.